extensions/amp-viz-vega/0.1/amp-viz-vega.js

```diff
@@ -114,7 +114,7 @@ export class AmpVizVega extends AMP.BaseElement {
 
   /** @override */
   onLayoutMeasure() {
-    const box = this.getLayoutBox();
+    const box = this./*OK*/ getBoundingClientRect();
```

Comment:
Same question here re. sync measurement

Reply:
Measurements in the `onLayoutMeasure` are technically free since they are called in a version of vsync.measure from resources-impl. But it's also a deprecated component.